apps/test/unit/templates/ButtonTest.js

```diff
@@ -4,7 +4,7 @@ import sinon from 'sinon'; // eslint-disable-line no-restricted-imports
 
 import Button from '@cdo/apps/templates/Button';
 
-import style from '@cdo/apps/templates/button.module.scss';
```

Comment:
nit: Should we also create `legacySharedComponents` folder in `test/unit` and move this test file there?